test/cpp/util/proto_file_parser.h

```diff
@@ -26,6 +26,12 @@
 #include "test/cpp/util/config_grpc_cli.h"
 #include "test/cpp/util/proto_reflection_descriptor_database.h"
 
+#if defined(_WIN32) && !defined(__CYGWIN__)
+#define PATH_SEPARATOR ";"
```

Comment:
Let's prefix with GRPC_CLI to avoid polluting global namespace, or maybe make it a static class member?

Reply:
I have renamed the macro to `GPRC_CLI_PATH_SEPARATOR`. I could have made it a static field of `grpc::testing::ProtoFileParser`, but this would have required calling `absl::StrFormat` for each of the help messages instead of string literal concatenation.